src/components/FileUploader/FileUploader.js

```diff
@@ -140,15 +140,14 @@ export class FileUploaderButton extends Component {
       <div
         role="button"
         tabIndex="0"
-        className={classes}
+        className={`bx--btn--${buttonKind} ${classes}`}
         onKeyDown={evt => {
           if (evt.which === 13 || evt.which === 32) {
             this.input.click();
           }
         }}>
         <label
-          className={`bx--btn bx--btn--${buttonKind}`}
-          tabIndex={tabIndex}
+          tabIndex={tabIndex || null}
```

Comment:
Given I believe the original `tabIndex` prop is for the button, would you want to move it to the container `<div>`?

Reply:
I'm not sure what the original `tabIndex` prop was for but I've gone ahead and made that change since it makes more sense than to put it on the `label`